ipv8/util.py

```diff
@@ -38,3 +40,8 @@ async def call_async(*args, **kwargs):
 
 def strip_sha1_padding(s: bytes):
     return s[12:] if s.startswith(b'SHA-1\x00\x00\x00\x00\x00\x00\x00') else s
+
+
+def get_class(class_or_function):
```

Comment:
This function is not used outside of `loader.py`, please relocate it to that file.

Reply:
Relocated :)